src/ServicePulse.Host/vue/src/components/failedmessages/MessageGroupList.vue

```diff
@@ -209,7 +209,7 @@ async function saveRetryGroup(group) {
   // We've started a retry, so increase the polling frequency
   changeRefreshInterval(1000);
 
-  saveDeleteNote(group, true);
+  await saveDeleteNote(group, true);
```

Comment:
this is changing behaviour to wait for the delete note promise to complete before continuing. If this isn't necessary then it will affect performance